common/views/themes/newConfig.ts

```diff
@@ -0,0 +1,333 @@
+  // khaki: { base: '#4f7d68', dark: '', light: '#9bc0af' },
+  // salmon: { base: '#ff6f59', dark: '', light: '#ff9585' },
+  // greys: {
+  //   2: '#323232', // 20
+  //   4: '#6b6b6b', // 42
+  //   5: '#8f8f8f', // 56
+  //   8: '#d9d9d9', // 85
+  //   9: '#e8e8e8', // 91
+  // },
+
+  //
+  // unchanged
+  white: { base: '#ffffff', dark: '', light: '' },
+  black: { base: '#121212', dark: '', light: '' },
+  // the dark value is legacy, do we change it?
+  red: { base: '#e01b2f', dark: '#c1192a', light: '' },
+  transparent: {
+    base: 'transparent',
+    dark: 'transparent',
+    light: 'transparent',
+  },
+  inherit: { base: 'inherit', dark: '', light: '' },
+  currentColor: { base: 'currentColor', dark: '', light: '' },
+
+  //
+  // existing names, new shade
+  purple: { base: '#724e91', dark: '', light: '#baa4cd' },
+  turquoise: { base: '#1dbebb', dark: '', light: '#a2eeed' },
+  // the dark value is legacy, do we change it?
+  green: { base: '#0b7051', dark: '#146a5c', light: '' },
+  // there's an even paler yellow, what do we do with it? (#fff9e6)
+  yellow: { base: '#ffce3c', dark: '', light: '#ffebad' },
+  cream: { base: '#d9d8d0', dark: '', light: '#edece4' },
+
+  //
+  // all new, used nowhere
+  blue: { base: '#27476e', dark: '', light: '#a4bfdf' },
+  khaki: { base: '#4f7d68', dark: '', light: '#9bc0af' },
+  salmon: { base: '#ff6f59', dark: '', light: '#ff9585' },
+
+  // greys
+  // pumice becomes cream
+  smoke: { base: '#e8e8e8', dark: '', light: '' },
+  pumice: { base: '#d9d8d0', dark: '', light: '' },
+  silver: { base: '#8f8f8f', dark: '', light: '' },
+  pewter: { base: '#6b6b6b', dark: '', light: '' },
+  // the dark value is legacy, can we get rid of it?
+  // Used once catalogue/webapp/components/IIIFViewer/IIIFViewer.tsx
+  charcoal: { base: '#323232', dark: '#2e2e2e', light: '' },
+
+  //
+  // old colours that don't have an equivalent in the new palette
+  // i've replaced them with some equivalents, but eventually go through codebase and change name
+
+  // teal: { base: '#006272', dark: '', light: '' },
+  // replaced with khaki?
+  teal: { base: '#4f7d68', dark: '', light: '' },
+
+  // not used anywhere
+  // cyan: { base: '#298187', dark: '', light: '' },
+
+  // orange: { base: '#e87500', dark: '', light: '' },
+  // replaced with salmon?
+  orange: { base: '#ff6f59', dark: '', light: '' },
+
+  // not used anywhere but PaletteColorPicker, so commented it out of it
+  // brown: { base: '#815e48', dark: '', light: '' },
+
+  // marble: { base: '#bcbab5', dark: '', light: '' },
+  // replaced with pumice/grey8
+  marble: { base: '#d9d9d9', dark: '', light: '' },
+
+  // what do we do with this one?
+  // The following 'black' is only to be used for the item viewer, does it need to be in theme?
+  coal: { base: '#1f1f1f', dark: '', light: '' },
+
+  // doesn't seem to be used anywhere but here common/views/components/Tasl/Tasl.tsx, does it need to be in theme?
```

Comment:
I vote for taking it out of the theme – it's very one-off-y

Reply:
Cool, I've done it now and replaced the one offs directly in their file (see new commit), so it'll be the case whether toggle is on or off but the value stays the same.